modules/33acrossBidAdapter.js

```diff
@@ -526,14 +578,19 @@ function _isIframe() {
 }
 
 // **************************** INTERPRET RESPONSE ******************************** //
-// NOTE: At this point, the response from 33exchange will only ever contain one bid
-// i.e. the highest bid
 function interpretResponse(serverResponse, bidRequest) {
   const bidResponses = [];
-
-  // If there are bids, look at the first bid of the first seatbid (see NOTE above for assumption about ttx)
-  if (serverResponse.body.seatbid.length > 0 && serverResponse.body.seatbid[0].bid.length > 0) {
-    bidResponses.push(_createBidResponse(serverResponse.body));
+  const { seatbid, cur } = serverResponse.body;
+
+  if (seatbid.length > 0) {
+    seatbid.forEach(function(seat) {
+      if (seat.bid.length > 0) {
+        seat.bid.forEach(function(bid) {
```

Comment:
is it possible for ttx to return a seatbid like this `[{}]`? if thats the case, maybe a check is needed for `seat.bid`

Reply:
This is defensive programming. Just in case we decide in future for 33XCH to return {} under a seat that doesn't generate winning bid, a response like so:
```
{
  seatbid: [
    {seat: 1},
    {seat: 2, bid: [{...}]}
  ]
}
```

Reply:
I have updated the above logic to make it more consistent with my approach and your suggestion.